### PR TITLE
feat(kubernetes/artifacts): reference material

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

**Comment by lwander** [comment]
closes https://github.com/spinnaker/spinnaker/issues/2177

**Comment by dorbin** [comment]
Closes #2177, but it looks like the "Services, Ingresses" and "NetworkPolicies" sections are still empty.

File: reference/artifacts/in-kubernetes/index.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Artifacts play an important role in the Kubernetes Provider V2. Everything from
+the manifests you deploy, to the Docker images or ConfigMaps they reference
+can be expressed or deployed in terms of artifacts.
+
```
Comment (dorbin) [inline reference/artifacts/in-kubernetes/index.md:13]: Do you want to add anything to [About Spinnaker Artifacts](https://www.spinnaker.io/reference/artifacts/) to reference this new doc? Maybe just an additional bullet to the top list of example artifacts?

File: reference/artifacts/in-kubernetes/index.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  ```json
+  {
+       "type": "kubernetes/configMap",
+       "name": "frontend-configs",
+       "location": "prod",
+       "version": "v001"
+  }
+  ```
+
+As described the [manifests as artifacts](#manifests-as-artifacts) section, a
```
Comment (dorbin) [inline reference/artifacts/in-kubernetes/index.md]: "As described *in* the..."

**Review by dorbin — APPROVED** [review]
As usual, I'll do some edits in a separate PR

**Review by dorbin — APPROVED** [review]

File: reference/artifacts/in-kubernetes/index.md (hunk 1 above)
Comment (lwander) [reply reference/artifacts/in-kubernetes/index.md:13]: Thinking about it, I feel we should keep it unlisted until we make the rest of the k8s v2 docs public. I'll file an issue & link to this comment to make sure we don't forget this among other places.

**Pull request merged**